brainiak/fcma/mvpa_voxelselector.py

```diff
@@ -38,96 +37,37 @@ class MVPAVoxelSelector:
     Parameters
     ----------
 
-    raw\_data: list of 4D array
-        each element of the list is the raw data of a subject,
-        in the shape of [x, y, z, t]
+    data: 4D array in shape [brain 3D + epoch]
+        contains the averaged and normalized brain data epoch by epoch.
+        It is generated by io.prepare_searchlight_mvpa_data
 
     mask: 3D array
 
-    epoch\_info: list of tuple (label, sid, start, end).
-        label is the condition labels of the epochs;
-        sid is the subject id, corresponding to the index of raw_data;
-        start is the start TR of an epoch (inclusive);
-        end is the end TR of an epoch(exclusive).
-        Assuming len(labels) labels equals the number of epochs and
-        the epochs of the same sid are adjacent in epoch_info
+    labels: 1D array
+        contains the labels of the epochs, extracted from epoch\_info
```

Comment:
`epoch_info` is no longer defined in this module. Make a reference to its definition in `.io` or, better yet, remove it, since it is superfluous.